packages/palette/src/themes/v2.tsx

```diff
@@ -0,0 +1,389 @@
+    black80: "#333",
+    /** 60% black, bold copy, lower in hierarchy  */
+    black60: "#666",
+    /** 30 black (dark grey), placeholder text only  */
+    black30: "#C2C2C2",
+    /** 10 black (grey), borders, divider lines, and grey button only */
+    black10: "#E5E5E5",
+    /** 5 black (light grey), backgrounds only */
+    black5: "#F8F8F8",
+    /** Full Blue. Calls to action, highlights, edits */
+    blue100: "#0A1AB4",
+    /** 10% of blue100 on white. Backgrounds */
+    blue10: "#E6E8F7",
+    /** Full copper. In consideration, transition, temporary */
+    copper100: "#A85F00",
+    /** 10% of copper100 on white. Backgrounds */
+    copper10: "#F6EFE5",
+    /** Full green. Success, approval, go */
+    green100: "#217C44",
+    /** 10% of green100 on white. Backgrounds */
+    green10: "#E9F2EC",
+    /** Full purple, secondary brand color. Should only used in time/transitions (on hover, active state), for highlighting vital text, and links.   */
+    purple100: "#6E1EFF",
+    /** 30 purple (light purple), avoid usage  */
+    purple30: "#D3BBFF",
+    /* 5 purple, highlight, accent */
+    purple5: "#F8F3FF",
+    /** Full red. Notification, error, stop */
+    red100: "#E82F1D",
+    /** 10% of red100 on white. Backgrounds */
+    red10: "#FDEAE8",
+    /** Replaced by copper100; hex updated here for consistency but copper100 should be used instead! */
+    yellow100: "#A85F00",
+    /** 30 yellow (light yellow), avoid future use */
+    yellow30: "#FAE7BA",
+    /** Replaced by copper10; hex updated here for consistency but copper10 should be used instead! */
+    yellow10: "#F6EFE5",
+    /** Full white */
+    white100: "#FFF",
+  },
+
+  fontFamily,
+
+  fonts: TEXT_FONTS,
+  fontSizes: TEXT_FONT_SIZES,
+  letterSpacings: TEXT_LETTER_SPACING,
+  lineHeights: TEXT_LINE_HEIGHTS,
+
+  // prettier-ignore
+  /** Media queries to work with in web  */
+  mediaQueries: {
+    xl: `(min-width: ${breakpoints.xl})`,
+    lg: `(min-width: ${breakpoints.lg}) and (max-width: ${parseInt(breakpoints.xl, 10) - 1})`,
+    md: `(min-width: ${breakpoints.md}) and (max-width: ${parseInt(breakpoints.lg, 10) - 1})`,
+    sm: `(min-width: ${breakpoints.sm}) and (max-width: ${parseInt(breakpoints.md, 10) - 1})`,
+    xs: `(max-width: ${parseInt(breakpoints.sm, 10) - 1})`,
+    /** Determines if the input device has the notion of hover, e.g. mouse. */
+    hover: `not all and (pointer: coarse), not all and (-moz-touch-enabled: 1)`,
+  },
+
+  // https://github.com/dragma/styled-bootstrap-grid#styled-bootstrap-grid
+  grid: {
+    /**
+     * Breakpoints for the Artsy grid,
+     * https://www.notion.so/artsy/Grid-e489a52e26bd4319b6ee7898044a8a53
+     *
+     * This version of `styled-bootstrap-grid` requires breakpoint
+     * values to be integers (not px literals)
+     */
+    breakpoints: unitlessBreakpoints,
+    container: {
+      padding: 0,
+    },
+    row: {
+      padding: 0,
+    },
+    col: {
+      padding: 0,
+    },
+  },
```

Comment:
I was thinking we could avoid using this in V3 trees, but am remembering now that the artwork grids use it. Wondering how much work it would take to drop this library. I suspect it wouldn't be too bad?

Reply:
I believe our every-day artwork grids use some kind of mad science, not a formal grid.

The API between the new `GridColumns` component and the old grid is close enough to where i think we could swap it out and remove the old one, but we're looking at ~41 files minimum. Not sure about volt, etc: 

![Uploading Screen Shot 2021-02-28 at 7.37.01 PM.png…]()